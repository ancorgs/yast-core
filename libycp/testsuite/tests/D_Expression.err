Parsed:
----------------------------------------------------------------------
"NON-TERMINAL expression"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
true
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
false
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
-5
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
5
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
-1.20000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
30000000000.0000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
5.00000000000000
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
"Hello world"
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
.
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
.current.path
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
[1, [2, nil]]
----------------------------------------------------------------------
Parsed:
----------------------------------------------------------------------
$[2:[nil, (4 + 5)], "Hallo":`Greetings]
----------------------------------------------------------------------
