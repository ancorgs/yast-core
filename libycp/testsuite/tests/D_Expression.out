("NON-TERMINAL expression")
(true)
(false)
(-5)
(5)
(-1.20000000000000)
(30000000000.0000)
(5.00000000000000)
("Hello world")
(.)
(.current.path)
([1, [2, nil]])
($[2:[nil, 9], "Hallo":`Greetings])
